### ISSUE TITLE
Can't update my meteor App after updating the meteor

### ISSUE THREAD

**Issue by saurabh21shukla** [meteor/meteor#7317]
Hi,
After updating the meteor once I run my app I start downloading the Meteor tools and between this downloading process. It will terminate and show the error.

E:\RND\meteardemo\Myapps\testapp>meteor update
Sorry, this project uses Meteor 1.3.2.4, which is not installed and could not be downloaded. Please check to make
sure that you are online.
   Downloading meteor-tool@1.3.2_4...        [==========================] 100% 0.0s

**Comment by abernix** [comment]
I'm guessing this is related to network connectivity.  Did this happen to you multiple times or did you just try once?

**Comment by Ben305** [comment]
I'm seeing many hanging downloads today, too.

**Comment by saurabh21shukla** [comment]
Thanks guys for your support but after some effort I got the solution. It's due to some meteor and npm packages of old version. I have solved this by using the following steps.

1. I have copied the packages file in .meteor directory and paste it other place or directory.
2. Uninstall meteor-node-stubs package from node module.
3. Reinstall meteor-node-stubs in project.
4. Create new meteor project and replace my .meteor directory with new projects .meteor directory.
5. Replace packages file which I have copied previously from my .meteor directory.
6. Now run meteor first I download the packages and run successfully.
